<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Persona</class>
 <widget class="QWidget" name="Persona">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Letrado: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Celular: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Correo electrónico: </string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lCorreo2"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lDomicilioL"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Domicilio real:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lCelular2"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lDNI"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lLetrado"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Nombrey Apellido:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>DNI/CUIT:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lDomicioR"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lNombre"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lCorreo1"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Domicilio legal: </string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Celular:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lCelular1"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Correo electrónico: </string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QPushButton" name="pbGuardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lNombre</tabstop>
  <tabstop>lDNI</tabstop>
  <tabstop>lDomicioR</tabstop>
  <tabstop>lCorreo1</tabstop>
  <tabstop>lCelular1</tabstop>
  <tabstop>lLetrado</tabstop>
  <tabstop>lDomicilioL</tabstop>
  <tabstop>lCorreo2</tabstop>
  <tabstop>lCelular2</tabstop>
  <tabstop>pbGuardar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
